clang/include/clang/AST/ExprConcepts.h

```diff
@@ -81,6 +82,7 @@ class ConceptSpecializationExpr final : public Expr, public ConceptReference {
 
   static ConceptSpecializationExpr *
   Create(const ASTContext &C, ConceptDecl *NamedConcept,
+         const ASTTemplateArgumentListInfo *ArgsAsWritten,
```

Comment:
This change breaks the API/ABI.  Can the patch be re-written without this change?

Reply:
How is this an ABI break?  I don't really see how this ends up modifying mangling at all, and I can't imagine this would change calling convention, does it?

Reply:
Ah, I see, we mean for libclang.  Unfortunately I don't think we can?  Its pretty much the 'point' to this patch...

That said, this was broken in 15 and has to do with a 'not yet claimed supported' feature (concepts), that perhaps it wouldn't be end of the world to just not merge this one.

Reply:
Ah, sure, I mistakenly thought the whole point was to remove that constructor, but its really the call from ::Create that we're concerned about.

Not sure how to get this PR updated, but I'll give it a shot.  Is that the way we'd fix this?  Presumably I can just push to [llvm-issue61801](https://github.com/llvm/llvm-project-release-prs/tree/llvm-issue61801), right?

Reply:
I think pushing to llvm-issue61801 will work.  You can also push to your fork and then comment 
/branch user/llvm-project/branch on the bug report.

Reply:
Ah, unfortunately I don't have permissions to update this PR, so I guess I'll have to do your latter idea.